features/aanschrijfwijze.feature

```diff
@@ -136,135 +141,622 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+        | geen adellijke titel/predikaat, geen voorvoegsel | VL GN           |
+        | geen adellijke titel/predikaat, voorvoegsel      | VL VV GN        |
+        | predikaat, geen voorvoegsel                      | PK VL GN        |
+        | predikaat, voorvoegsel                           | PK VL VV GN     |
+        | adellijke titel, geen voorvoegsel                | VL AT GN        |
+        | adellijke titel, voorvoegsel                     | VL AT VV GN     |
+
+    Abstract Scenario: Alleenstaande, <titel>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende naam gegevens
+      | naam                                 | waarde            |
+      | voorvoegsel (02.30)                  | <voorvoegsel>     |
+      | geslachtsnaam (02.40)                | <geslachtsnaam>   |
+      | voornamen (02.10)                    | <voornamen>       |
+      | aanduiding naamgebruik (61.10)       | E                 |
+      | adellijke titel of predikaat (02.20) | <titel/predikaat> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | naam.aanschrijfwijze            |
+      Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+      | naam            | waarde            |
+      | aanschrijfwijze | <aanschrijfwijze> |
+
+      Voorbeelden:
+      | voorvoegsel | geslachtsnaam | voornamen | titel/predikaat | aanschrijfwijze         | titel                                            |
+      |             | Groenen       | Franklin  |                 | F. Groenen              | geen adellijke titel/predikaat, geen voorvoegsel |
+      | In het      | Veld          | Henk      |                 | H. In het Veld          | geen adellijke titel/predikaat, voorvoegsel      |
+      |             | Groenen       | Franklin  | G               | F. graaf Groenen        | adellijke titel, geen voorvoegsel                |
+      |             | Groenen       | Franklin  | JH              | Jonkheer F. Groenen     | predikaat, geen voorvoegsel                      |
+      | In het      | Veld          | Henk      | B               | H. baron In het Veld    | adellijke titel, voorvoegsel                     |
+      | In het      | Veld          | Henk      | JH              | Jonkheer H. In het Veld | predikaat, voorvoegsel                           |
+
+  Rule: De aanschrijfwijze voor een persoon met partner zonder adellijke titel/predikaat wordt als volgt samengesteld:
+        |                                                                            | naamgebruik | aanschrijfwijze   | opmerking |
```

Comment:
naamgebruik heeft in onze API aanduidingNaamgebruik en in LO GBA (de bron) "Aanduiding naamgebruik" (61.10)